React-Projects/job-portal/src/styles/Header.scss

```diff
@@ -0,0 +1,112 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  background-color: rgb(248, 248, 248);
+}
+
+header.nav-header {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  background-color: white;
+  color: black;
+  padding: 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(49, 45, 45, 0.1);
+  z-index: 1000;
+
+  .nav-wrapper {
+    display: flex;
+    align-items: center;
+
+    img {
+      height: 50px;
+      width: 130px;
+    }
+  }
+
+  .nav-button-wrapper {
+    .sign-up-button {
+      margin-right: 20px;
+      background-color: purple;
+    }
+
+    .login-button {
+      background-color: darkblue;
+    }
+
+    .sign-up-button:hover,
+    .login-button:hover {
+      opacity: 0.3;
+    }
+  }
+}
+
+nav ul {
```

Comment:
@codal-kcolaco Please avoid giving styles directly to tags. always use classname to it.

Reply:
@abhi-codal Will make sure from now on